src/templates/ssr/fabric/index.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
I tried to delete this file as we are not using it for Fabric Standard but it gives me an error. Not sure if there is a way to delete it without any compiling errors. I also added `export {}` because it wasn't passing "checking TypeScript" step in CI/CD.

Reply:
After moving the directory to be a CSR I was able to remove the `ts` file as now all the TypeScript code is in `index.svelte`